keymaps/tada68_default.json

```diff
@@ -0,0 +1,17 @@
+{
+  "keyboard": "tada68",
+  "keymap": "default",
+  "layout": "KEYMAP_ANSI",
```

Comment:
Change `KEYMAP_ANSI` to `LAYOUT_ansi` here.

The name of the key matrix was changed after you submitted this PR.